src/Search.php

```diff
@@ -1072,7 +1071,7 @@ public function add_load_more( $args = null ) {
 		$this->context['load_more'] = $args ?? [
 			'posts_per_load' => self::POSTS_PER_LOAD,
 			// Translators: %s = number of results per page.
-			'button_text'    => sprintf( __( 'SHOW %s MORE RESULTS', 'planet4-master-theme' ), self::POSTS_PER_LOAD ),
+			'button_text'    => sprintf( __( 'Show %s more results', 'planet4-master-theme' ), self::POSTS_PER_LOAD ),
```

Comment:
Shouldn't that be capitalized? Like `Show %s More Results` ?

Reply:
This is enforced by the CSS rule `text-transform: capitalize;` so we don't need to capitalize it here